scripts/multi_build.bash

```diff
@@ -16,9 +16,9 @@ declare -ri n="${n:-3}"
 declare -r ip_start="${ip_start:-127.0.0.1}"
 declare -r subnet="${subnet:-16}"
 declare -r ip_range="$ip_start/$subnet"
-declare -r entry="${entry:-main}" # you may use main_profile here to enable profiling
+declare -r l_entry="${l_entry:-main}" # you may use main_profile here to enable profiling
```

Comment:
Why the rename?

Reply:
It collides with $entry environment variable already used in Travis.